### ISSUE TITLE
Support for subsampled variable rate shading on ARM mobile GPUs

### ISSUE THREAD

**Issue by speedym** [baldurk/renderdoc#2598]
## Description

Vulkan extension VK_EXT_fragment_density_map is not supported by RenderDoc when only subsampled textures and render targets are supported by the mobile device, as is the case on ARM based mobile devices. It would be great if RenderDoc supported such devices, and displayed either upsampled content of the textures or even raw contents of the subsampled textures, like XCode frame capture does for the Metal API / iOS devices & VRS.

## Environment

* RenderDoc version: 1.20
* Operating System: Linux
* Graphics API: Vulkan

**Comment by tuxerr** [comment]
This is on my short-term radar and I'm working on it. If you have knowledge of anybody else actively contributing to the effort, please let me know @speedym

**Comment by JoseEmilio-ARM** [comment]
Hi @tuxerr , would you have any updates on this?

**Comment by oscarbg** [comment]
@JoseEmilio-ARM yep, hope is coming soon in new "Renderdoc for ARM GPUs" and later via ARM contributions to Renderdoc..
